app/policies/article_policy.rb

```diff
@@ -1,10 +1,10 @@
-class ArticlePolicy
+class ArticlePolicy < BasePolicy
   def initialize(user, article)
-    @user = user
+    self.user = user
```

Comment:
I use this policy only in views, instead of 'if current_user == user'. I can use simple view helper instead of this policy?